components/prism-php.js

```diff
@@ -42,14 +49,19 @@
 		}
 	});
 
-	Prism.languages.insertBefore('php', 'keyword', {
+	// Must be defined before class-name (because primitive types must be matched before class names in function typehint)
+	Prism.languages.insertBefore('php', 'class-name', {
 		'variable': /\$+(?:\w+\b|(?={))/i,
 		'package': {
-			pattern: /(\\|namespace\s+|use\s+)[\w\\]+/,
+			pattern: /(\\|namespace\s+|use\s+(function\s+)*)[a-zA-Z_][\w\\]+/,
```

Comment:
Keywords are case insensitive, so please add the `i` flag.

> `(function\s+)*` 

Star? Shouldn't that be a `?`?

> `[a-zA-Z_][\w\\]+`

Why the `[a-zA-Z_]`? This prevents the pattern from matching text like `use \bar` in full.

Reply:
Yes star changed to `?`
My point was to not match names beginning with number (that is syntax error) -> fixed in next commit